### PR TITLE
Add build script and update instructions to test locally

### PR DESCRIPTION
**Pull request by mehulkar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "build": "pnpm -- turbo run build --filter=docs",
+    "build": "pnpm -- turbo run build",
```
Comment (mehulkar) [inline package.json:6]: Vercel settings are not using this, so it's safe to change. Vercel UI already has a custom build command

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,14 +7,17 @@ Thanks for your interest in contributing to Turbo!
...
+      - [Rust Tests](#rust-tests)
   - [Debugging Turborepo](#debugging-turborepo)
   - [Benchmarking Turborepo](#benchmarking-turborepo)
   - [Updating `turbo`](#updating-turbo)
   - [Publishing `turbo` to the npm registry](#publishing-turbo-to-the-npm-registry)
   - [Contributing to Turbopack](#contributing-to-turbopack)
+    - [Turbopack Architecture](#turbopack-architecture)
     - [Testing Turbopack](#testing-turbopack)
     - [Benchmarking Turbopack](#benchmarking-turbopack)
+    - [Profiling Turbopack](#profiling-turbopack)
```
Comment (mehulkar) [inline CONTRIBUTING.md:20]: TOC was updated when I saved the file. I don't know what plugin I have that is doing this magic.

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "build": "pnpm -- turbo run build --filter=docs",
+    "build:turbo": "cd cli && make && cd ..",
```
Comment (tknickman) [inline package.json:7]: I would do: 

```suggestion
    "build:turbo": "pnpm --filter=cli build",
```

Simpler and avoids the `cd`

Comment (mehulkar) [reply package.json:7]: good idea! done

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -36,14 +39,20 @@ Building
...
+- run unit tests with `pnpm run --filter=cli test`
+- run integration tests with `pnpm run --filter=cli integration-tests`
+- run e2e tests with `pnpm run --filter=cli e2e`
 
-To run a single test, you can run `go test ./[path/to/package/]`. See more [in the Go docs](https://pkg.go.dev/cmd/go#hdr-Test_packages).
+To run a single Go test, you can run `go test ./[path/to/package/]`. See more [in the Go docs](https://pkg.go.dev/cmd/go#hdr-Test_packages).
+
+#### Rust Tests
+
+You can run `cargo test -p turborepo-lib`, or [install nextest](https://nexte.st/book/pre-built-binaries.html)
```
Comment (ForsakenHarmony) [inline CONTRIBUTING.md:54]: nextest runs faster, I think it should be recommended first and `cargo test` as the backup option

**Review by ForsakenHarmony — APPROVED** [review]

**Comment by mehulkar** [comment]
Node.js linting does not seem related to this PR. Merging

**Pull request merged**